FieldLow: [103, 0, 0]
ballLow: [0, 0, 132]
ballHigh: [255, 255, 255]
obstLow: [0, 0, 0]
obstHigh: [0, 0, 0]
goalLow: [0, 0, 0]
goalHigh: [0, 0, 0]
amount_of_white_neighbours: 1
resolution_of_digestion: 20
radius_of_major_nodes: 5
nearest_neighbour_scope: 15
line_traversal_scope: 2
Tolerance: 5